lib/utils/log-utils.js

```diff
@@ -0,0 +1,206 @@
+
```

Comment:
Thanks for all this work, @npetruzzelli! But I think you heavily overdo it. We don't really need a new powerful logging utility here. It is enough to make one simple helper function in the same file (similar to what I presented in the issue) and use this utility in 3 places in `parseConfig` function where we currently have `log.error() + process.exit(1)` calls.

This warning logic is only needed temporarily and it is written to be removed, so it does not need to be flexible or reusable or anything like that.

Reply:
Got it. At the very least I learned a bit more about the currently logging works. I'll update the PR using your comment in the issue as a template. I'll needed to change it a little bit to account for:

-   the one instance where multiple arguments are logged
-   the import failure logging multiple times before exit